docs/usermanual.md

```diff
@@ -544,3 +545,15 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the IOT Agent (as explained [here](installationguide.md#usage) and then you
```

Comment:
Travis CI detects the following problem regarding this line:

  551:49  ✓ error  Incorrect usage of the term: “IOT”, use “IoT” instead  terminology

Please, fix it.

(Full log: https://travis-ci.org/telefonicaid/iotagent-ul/jobs/639955851?utm_medium=notification&utm_source=github_status)

Reply:
fixed in 83450903a8c2896246a015ebe22c3f59469d3e7d